test/controllers/medias_controller_test.rb

```diff
@@ -7,23 +7,6 @@ def setup
     @controller = Api::V1::MediasController.new
   end
 
-  test "should return error if url is not provided" do
-    authenticate_with_token
-    get :index, format: :json
-    assert_response 400
-  end
-
-  test "should return error if not authenticated" do
-    get :index, url: 'http://meedan.com', format: :json
-    assert_response 401
-  end
-
-  test "should parse media" do
-    authenticate_with_token
-    get :index, url: 'http://twitter.com/meedan', format: :json
-    assert_response :success
-  end
-
```

Comment:
@daniel-shuy why did you remove these tests? They were not affected by your changes, they are also passing.
Could you add them back, please?

Reply:
I've moved them to the RSpec Rails test (see https://github.com/meedan/pender/pull/233/files#diff-b2c51c61291a55b9a9e7673b205ebc31d12b9217687734e377fb6394d33cbdfc)